static/app/components/metrics/mriSelect/metricListItemDetails.tsx

```diff
@@ -207,12 +205,7 @@ export function MetricListItemDetails({
                 return (
                   <Fragment key={tag.key}>
                     <TagWrapper>
-                      <Button
```

Comment:
Shouldn't we do the same here? Only disable those that are not available in the time range?

Reply:
hmm, right. What is even more important, at this point the metric is most likely not yet selected. Therefore, we do not have a selected condition and aggregate that we could use for resolving.
k, let's keep it disabled 👍  But only for virtual metrics pls.

Reply:
Yes, that was my idea also, only tags from virtual metrics might be disabled (have `isQueryable` field set to false). That check is happening in https://github.com/getsentry/sentry/pull/74134/files#diff-24ee1ab39c06e08aaf8bf035fcbd7d2256f8b056daf112bf8ec9070b94982dafR90-R107

Reply:
but in here is no check?

Reply:
Conditional "clickability" added